org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -157,14 +262,33 @@ public override bool isHoverActive
                     bool hoverActive = base.isHoverActive;
                     if (hoverActive)
                     {
-                        if (xrController is ArticulatedHandController handController)
+                        bool isPalmFacingAway = false;
+
+                        #pragma warning disable CS0618 // Type or member is obsolete
+                        #pragma warning disable CS0612 // Type or member is obsolete
```

Comment:
Limit the scope of the warning disable...it should only cover controller usage, and nothing else.  PLease follow the pattern I used in my draft PR 

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/745/files

Reply:
done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/750/commits/17305f6a751cc7715f888db852aaebace17f6dd9